src/main/java/uk/gov/hmcts/divorce/payment/PaymentService.java

```diff
@@ -39,6 +80,63 @@ public OrderSummary getOrderSummary() {
             .build();
     }
 
+    public PbaResponse processPbaPayment(CaseData caseData, Long caseId) {
```

Comment:
It would be nice is this didn't take the whole case data. Is it possible to just pass the OrderSummary for the thing that we're trying to take payment for? I'm thinking that in future we might have other things to pay for like amends, in which case we'll need to send a different OrderSummary

Reply:
payment request requires data which is in case data https://github.com/hmcts/nfdiv-case-api/pull/623/files#diff-86d92efa4d934715621e6205346aa501f609e65f6665b0ccdd3b6dad23de70faR213-R239
 so we will need to pass case data anyways even if it is amend or create.